CONTRIBUTING.md

```diff
@@ -70,19 +78,40 @@ Git even has a `-s` command line option to append this automatically to your com
 git commit -s -m 'This is my commit message'
 ```
 
-Each Pull Request is checked  whether or not commits in a Pull Request do contain a valid Signed-off-by line.
+**For GPG Signatures:**
+
+GPG signatures verify the identity of the committer. For detailed setup instructions, see GitHub's documentation on [signing commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).
+
+Quick setup:
+
+```sh
+git config --global user.signingkey YOUR_KEY_ID
+git config --global commit.gpgsign true
+```
+
+**For both DCO sign-off and GPG signature in one command:**
 
-#### I didn't sign my commit, now what?!
+```sh
+git commit -S -s -m 'This is my signed and signed-off commit message'
+```
+
+For detailed instructions on setting up both signature types, see [docs/commit-signing.md](./docs/commit-signing.md).
+
+Each Pull Request is checked to ensure all commits contain valid DCO sign-offs and GPG signatures.
+
+#### I didn't sign my commit, now what?
 
 No worries - You can easily replay your changes, sign them and force push them!
 
+**For adding both DCO sign-off and GPG signature:**
+
 ```sh
 git checkout <branch-name>
-git commit --amend --no-edit --signoff
+git commit --amend --no-edit -S -s
 git push --force-with-lease <remote-name> <branch-name>
 ```
 
-*Credit: This doc was cribbed from Dapr.*
+For more detailed instructions on fixing commits, see [docs/commit-signing.md](./docs/commit-signing.md).
```

Comment:
remove the commit signing doc and point to https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits instead.  Don't drop the `credit comment`